app/renderer/components/preferences/payment/disabledContent.js

```diff
@@ -59,7 +58,7 @@ const styles = StyleSheet.create({
     backgroundColor: globalStyles.color.lightGray,
     borderRadius: globalStyles.radius.borderRadiusUIbox,
     padding: '40px',
-    fontSize: paymentStyles.font.regular,
+    fontSize: globalStyles.payments.fontSize.regular,
```

Comment:
To remove `paymentsStyles` for now. It is not maintained well and moving the values to global.js makes it the code much more understandable.

I'm going to re-introduce `paymentsStyles` after files related with about:preferences#payments are polished and the other stuff (which could be related with them) on other files than ones under `payment` is also converted to normalize the common styles, if any.